apps/arbeidsforhold-api/src/test/java/no/nav/registre/testnorge/arbeidsforhold/ApplicationContextTest.java

```diff
@@ -20,6 +20,6 @@
      */
     @Test
     @SuppressWarnings("java:S2699")
-    public void load_app_context() {
+    public void load_app_context2() {
```

Comment:
Det var kanskje denne funksjonen som fikk nytt navn da du hadde demo for oss? Kan kanskje fjerne `2` fra navnet?

Reply:
Jupp :D